source/extended-json/extended-json-query.md

```diff
@@ -0,0 +1,56 @@
+# Extended JSON Query
+
+## Abstract
+
+Extended JSON query variation is a superset of Extended JSON. It is
+intended to be used for serializing queries that can be specified in
+the mongo shell. It supplements the Extended JSON syntax with support
+for the following features:
+
+- Trailing commas after the last value in an object
+
+- NumberLong values in quotation marks (aka "double quote")
+  - Apostrophe (aka "single quote") is not supported
+
+- Dates constructed with `new Date()` including an `ISODate` format
+  - Including support for pre-epoch dates
+
+- Regex pattern enclosed in slashes
+
+## Examples
+
+### Trailing Comma
+    The following example shows an example of a JSON object with a trailing comma after the `b: 2` element. The trailing comma is supported only in objects, not in arrays. This is convenient when manually editing the text representation.
```

Comment:
How can I reformat this to satisfy the linter? The text should not be considered a code block.

The linter suggests:
```diff
 ### Trailing Comma
-    The following example shows an example of a JSON object with a trailing comma after the `b: 2` element. The trailing comma is supported only in objects, not in arrays. This is convenient when manually editing the text representation.
+
+```
+The following example shows an example of a JSON object with a trailing comma after the `b: 2` element. The trailing comma is supported only in objects, not in arrays. This is convenient when manually editing the text representation.
+```
+
```